plugins/modules/vm_nic.py

```diff
@@ -140,37 +136,6 @@
 """
 
 RETURN = r"""
-record:
-  description:
-    - The created or changed record for nic on a specified virtual machine.
-  returned: success
-  type: dict
-  contains:
-    uuid:
-      description: Unique identifier
-      type: str
-      sample: 07a2a68a-0afa-4718-9c6f-00a39d08b67e
-    vlan:
-      description: VLAN tag of the interface
-      type: int
-      sample: 15
-    type:
-      description: Virtualized network device types
-      type: str
-      sample: virtio
-    mac:
-      description: MAC address of the virtual network device
-      type: str
-      sample: 12-34-56-78-AB
-    connected:
-      description: Enabled and can make connections
-      type: bool
-      sample: true
-    ipv4_addresses:
-      description: IPv4 addresses registered with this device
-      type: list
-      elements: str
-      sample: 192.0.2.1
 records:
```

Comment:
Here the "type" needs to be list? Also the second line under "description" is not needed anymore

Reply:
List is ok, a single NIC can have multiple IP addresses.

Reply:
Hm, I can see now that it is not clear that this question is meant for "records"

Reply:
Removed the second line from "description", thank you for noticing.